packages/aws-cdk/lib/api/aws-auth/sdk.ts

```diff
@@ -1,3 +1,71 @@
+const {
```

Comment:
Import over Require support was added in `v0.25.3` introduced in https://github.com/awslabs/aws-sdk-js-codemod/pull/652